### ISSUE TITLE
Add support for TimeWindow Compaction Strategy

### ISSUE THREAD

**Issue by hasmikhail** [scylladb/scylla#1432]
extracted from PR #511 

"DTCS will be deprecated, suggest support TWCS , see https://issues.apache.org/jira/browse/CASSANDRA-9666"

cassandra implementation https://github.com/jeffjirsa/cassandra/commit/f966b5be96dd91e81d2e081d9e604c8ae6a0f082

**Comment by qrpike** [comment]
@hasmikhail Are there any updates on this? I am using scylla 2.0.1 and when I try to alter my table to use TWCS I get that the strategy isnt found.

eg: 
```cql
ALTER TABLE prices WITH compaction = {
	'compaction_window_size': '30', 
	'compaction_window_unit': 'DAYS', 
	'class': 'TimeWindowCompactionStrategy'};
```

I get: `ConfigurationException: Unable to find compaction strategy class 'TimeWindowCompactionStrategy'`

**Comment by slivne** [comment]
TimeWindowCompactionStrategy will be part of 2.1

**Comment by raphaelsc** [comment]
On Tue, Nov 28, 2017 at 5:46 AM, Quinton Pike <notifications@github.com>
wrote:

> @hasmikhail <https://github.com/hasmikhail> Are there any updates on
> this? I am using scylla 2.0.1 and when I try to alter my table to use TWCS
> I get that the strategy isnt found.
>

TWCS will be first available in scylla 2.1

**Comment by qrpike** [comment]
@slivne Any rough estimates on 2.1 release date?

Thanks for the great work on this project!
